client/containers/Pub/SpubHeader/SpubHeaderTab.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useRef, useState, useEffect } from 'react';
+
+import { useViewport } from 'client/utils/useViewport';
+
+require('./spubHeaderTab.scss');
+
+type Props = {
+	children?: React.ReactNode;
```

Comment:
No reason to make `children` optional here — if you don't have any, you won't want to use this component!